Source/WebCore/html/HTMLElement.cpp

```diff
@@ -1489,6 +1489,10 @@ ExceptionOr<bool> HTMLElement::togglePopover(std::optional<bool> force)
         auto returnValue = showPopover();
         if (returnValue.hasException())
             return returnValue.releaseException();
+    } else {
+        auto check = checkPopoverValidity(*this, popoverData() ? popoverData()->visibilityState() : PopoverVisibilityState::Showing);
```

Comment:
```suggestion
        auto check = checkPopoverValidity(*this, popoverData() ? popoverData()->visibilityState() : PopoverVisibilityState::Hidden);
```

I think this makes more sense given the argument represents the current state. Although it doesn't really make a difference in this case :)

Reply:
A good point, indeed I figured that the value did not matter because of the first check in checkPopoverValidity, but indeed Hidden is more intuitive. Can fix before landing, but let's see if the spec change is agreed upon as-is first.